Description: Patch the dependency on is-absolute with path-is-absolute
Forwarded: not-needed
Author: Paolo Greppi <paolo.greppi@libpf.com>

Index: node-parse-filepath/index.js
===================================================================
--- node-parse-filepath.orig/index.js
+++ node-parse-filepath/index.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var path = require('path');
-var isAbsolute = require('is-absolute');
+var isAbsolute = require('path-is-absolute');
 var pathRoot = require('path-root');
 var MapCache = require('map-cache');
 var cache = new MapCache();
